php/libraries/Database.class.inc

```diff
@@ -1159,5 +1159,83 @@ class Database
         }
     }
 
+    /**
```

Comment:
I have no idea about this or how it really works. 

I think Justin wrote it. We can try removing it but I'm not sure how it would impact the logic in the genomic_uploader